spec/models/user_spec.rb

```diff
@@ -227,50 +197,21 @@
       end
     end
 
-    it "should set requires_manual_approval if the user is an affiliate and the email is not government_affiliated" do
-      %w( aff@agency.COM aff@anotheragency.com admin.gov@agency.org anotheradmin.MIL@agency.ORG escape_the_dot@foo.xmil ).each do |email|
+    it 'should set requires_manual_approval if the user is an affiliate and the email is not government_affiliated' do
+      %w[aff@agency.COM aff@anotheragency.com admin.gov@agency.org anotheradmin.MIL@agency.ORG escape_the_dot@foo.xmil].each do |email|
```

Comment:
As long as you're cleaning this up, it's a good time to break up this long line:
```ruby
%w[aff@agency.COM
   aff@anotheragency.com 
...
```